src/ui/LinkPointer.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+  import Svg from '@/ui/Svg/svelte'
+
+  let className = ''
+  export { className as class }
+  export let href: string
+  export let label: string
+</script>
+
+<a {...$$props} {href} class="link-pointer v-center {className}">
+  {label}
+  <Svg id="right-arrow" w={10} h={5} class="$style.svg" />
```

Comment:
Not all links have right-arrow

Reply:
Task has 2 parts:
https://www.notion.so/santiment/Link-buttons-updates-ddb3cd3c912341a880e46a17b302bd35

Reply:
See `stories/LinkPointer.stories.svelte`